include/wlr/backend.h

```diff
@@ -57,5 +57,10 @@ struct wlr_session *wlr_backend_get_session(struct wlr_backend *backend);
  * Returns the clock used by the backend for presentation feedback.
  */
 clockid_t wlr_backend_get_presentation_clock(struct wlr_backend *backend);
+/**
+ * Returns the file descriptor that is internally opened for rendering.
```

Comment:
"rendering" seems a little bit off. In the future the backend will be completely decoupled from the renderer and any rendering operation.

The DRM backend uses this FD for KMS operations. Other backends don't really use the FD, it's just relayed as a hint for the compositor's renderer, so that wlroots and the parent X11 server or Wayland compositor can use the same device.

Maybe "used by the backend's underlying platform" would be more accurate?